modules/jumpbox/jump-sg.tf

```diff
@@ -0,0 +1,41 @@
+#==============================================================
+# Jumpbox / jump-sg.tf
+#==============================================================
+
+# Security groups for Jumpbox
+
+resource "aws_security_group" "jump_ssh_sg" {
+  # Allow SSH to jump host
+  name = "jump_ssh"
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["${var.ssh_ip_address}"]
+  }
+
+  ingress {
```

Comment:
Why does the jumpbox need to ssh to itself?

Reply:
I had a loop and I think I put that in because I misunderstood something; I've taken it out.